src/main/java/com/softserveinc/dokazovi/repositories/UserRepository.java

```diff
@@ -15,4 +18,12 @@
 	@Query(nativeQuery = true,
 			value = "SELECT * FROM users u WHERE u.status='ACTIVE' ORDER BY random()")
 	Page<UserEntity> findRandomActiveUsers(Pageable pageable);
+
+	@Query(nativeQuery = true,
+			value = "SELECT * FROM users u "
+						+ "WHERE u.status='ACTIVE' "
+						+ "AND u.direction_id IN :#{#directionsIds}"
```

Comment:
If you type "AND u.direction_id IN (:directionsIds)" you will get the same behavior